kvui.py

```diff
@@ -5,6 +5,9 @@
 import re
 from collections import deque
 
+if __debug__:
+    assert "kivy" not in sys.modules, "kvui should be imported before kivy for frozen compatibility"
```

Comment:
```suggestion
assert "kivy" not in sys.modules, "kvui should be imported before kivy for frozen compatibility"
```
The `if __debug__` is not necessary. All `assert` statements are removed in the optimized frozen build.

Reply:
that's what i get for thinking a three liner is simple enough after optimizing the `if "kivy" not in sys.modules:` to just be on the assert